src/frontend/Sider/Behandling/VedtakOgBeregning/Læremidler/InnvilgeVedtak/Beregningsresultat.tsx

```diff
@@ -32,6 +32,7 @@ export const Beregningsresultat: FC<{ beregningsresultat: BeregningsresultatLær
             <Label>Prosent</Label>
             <Label>Månedsbeløp</Label>
             <Label>Stønadsbeløp</Label>
+            <div></div>
```

Comment:
Tror man kan lukke den sånn her og
```suggestion
            <div />
```